spec/models/automation_request_spec.rb

```diff
@@ -119,7 +119,7 @@
   end
 
   context ".create_from_scheduled_task" do
-    let(:admin) { FactoryBot.create(:user_miq_request_approver) }
+    let(:admin) { FactoryBot.create(:user_miq_request_approver, :userid => 'admin') }
```

Comment:
Same here...

Reply:
@bdunne @kbrock Does something need to be changed here, or can we do it in a followup PR?

Reply:
Thanks for the ping here

This is creating a super duper admin but with different privileges. That should be a red flag.

don't set the userid here.